src/components/Card/index.js

```diff
@@ -88,14 +88,21 @@ const bases = {
   large: '2/3',
 };
 
-export const Card = ({ category, content, width = 'medium', link, image }) => (
+export const Card = ({
+  category,
+  content,
+  width = 'medium',
+  link,
+  image,
+  source,
```

Comment:
Rather than make Card know about source types you might consider making it easy to override basis by adding `...rest` here and then adding `{...rest}` between lines 105 and 106. Then you could pass `basis="auto"` in the events page instead of source.

Reply:
Thanks @MikeKingdom! I made the update here 81aacb4 and is working 👍